### DISCUSSION TITLE
Author profile does not show links

### DISCUSSION THREAD

**Discussion by rubinovk** [mmistakes/minimal-mistakes#3082] [Q&A] [answered]
- [] This is a question about using the theme.
- [X] I believe this to be a bug with the theme --- not Jekyll, GitHub Pages or one of the bundled plugins.
- [ ] This is a feature request.
- [X] I have updated all gems with `bundle update`.
- [X] I have tested locally with `bundle exec jekyll build`.

---
## Environment informations
- **Minimal Mistakes version: v.3.3.6** 
- **`github-pages` or `jekyll` gem version: 89** 
- **Operating system: OSX** 

---
## Expected behavior

As per Configuration and https://mmistakes.github.io/minimal-mistakes/docs/layouts/#author-profile 

``` markdown
author:
  name   : "Your Name"
  avatar : "bio-photo.jpg"  # placed in /images/
  bio    : "My awesome biography constrained to a sentence or two goes here."
  email  : # optional
  uri    : "http://your-site.com"
```

should make the author uri visible with an icon, as well as others, if filled in.
## Actual behavior

Non of the possible authorprofile links is visible. Neither in the https://mmistakes.github.io/minimal-mistakes/about/ example sidebar that should have author uri  `uri :"https://mademistakes.com"`

![screen shot 2016-08-03 at 14 13 30](https://cloud.githubusercontent.com/assets/2063487/17365108/82172a1c-5984-11e6-87b1-ec85c047b625.png)

Thanks for the great theme!

**Comment by rubinovk** [comment]
After additional testing found that it's my browser's AdBlock that disables these links. Why would it? :$

**Comment by mmistakes** [comment, marked as answer]
It's a known issue with adblockers and Font Awesome. The links are there, you're just not seeing them because your adblocker is removing them.

Here's the upstream issue if you want more context. https://github.com/FortAwesome/Font-Awesome/issues/1799

**Comment by rubinovk** [comment]
Thanks for your quick reply and the context.

**Comment by proof-by-accident** [comment]
Hi I'm actually having a similar problem- my social media links are showing up, but the FontAwesome icons are not.

_config.yml:
`author:                                                                                               
  name: proof-by-accident                                                                             
  avatar: "/images/prof_pic.jpg"                                                                      
  twitter: ProofByAccident                                                                            
  github: proof-by-accident                                                                           
  reddit: proof_by_accident  `

I added the Reddit link following the "Layout Customization" page, but no icons appeared.  Do I have to download the icon images and store them somewhere specific or something?

**Comment by mmistakes** [comment]
@proof-by-accident No idea. If you share a link to a public repo and the site I can maybe troubleshoot. Otherwise it's impossible for me to tell what might have went wrong for you.

**Comment by proof-by-accident** [comment]
Disregard this, I was able to fix the error by moving all the _includes from the gem directory to my blog directory.  Thanks for your fast response!

**Comment by PrasanthJayaraman** [comment]
> It's a known issue with adblockers and Font Awesome. The links are there, you're just not seeing them because your adblocker is removing them.
> 
> Here's the upstream issue if you want more context. [FortAwesome/Font-Awesome#1799](https://github.com/FortAwesome/Font-Awesome/issues/1799)

Yes adblocker is the issue in my case..

**Comment by HabibRh26** [comment]
using ubuntu18 and chrome as browser. It is not displaying. How Can I see the actual author profile output?

**Comment by Destaq** [comment]
Is there any workaround to this without having users disable their adblock social lists?

**Comment by HabibRh26** [comment]
@Destaq I found that the default code is commented out.I just uncomment those and it worked fine in my local machine(os-ubuntu18, browser-chrome). Nothing related to adblocker or anything

**Comment by Destaq** [comment]
@HabibRh26 Could you please explain how you did that? I'm not sure what 'default code' it is that you are referencing.

**Comment by HabibRh26** [comment]
default code meaning when you download the theme then the codes are commented with #,uncomment those and run, Hope it will work

**Comment by gimquokka** [comment]
wow... add block... why I didn't know that.... TT
